etcdctl/README.md

```diff
@@ -1174,11 +1159,118 @@ The provided transformer should read until EOF and flush the stdout before exiti
 # Role roleA is revoked from user userA
 ```
 
+### USER DELETE \<user name\>
+
+`user delete` deletes a user.
+
+#### Return value
+
+##### Simple reply
+
+- `User <user name> deleted`. Exit code is zero.
+
+- Error string if failed. Exit code is non-zero.
+
+#### Examples
+
+```bash
+./etcdctl --user=root:123 user delete myuser
+# User myuser deleted
+```
+
+### AUTH \<enable or disable\>
```

Comment:
I don't have strong preference on this but @sinsharat 's opinion seems to be reasonable.